tests/snmpsim/racomos.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|Microwave Link
```

Comment:
Is this the genuine sysDescr from a device?

Reply:
not either RAY - Microwave Connection or Microwave Link I'm not done yet, I'm testing it

Reply:
Ok so this PR isn't ready?

Reply:
no, I needed to solve the problems mentioned above

Reply:
Ok